### PR TITLE
Ww 362 updated i18n

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
@Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/crowdin-autocreate.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [crowdin.project]
-projectid = "ext-createnewwiki"
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/crowdin-autocreate.conf:2]: why this change?

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/CreateNewWiki/crowdin-autocreate.conf (hunk 1 above)
Comment (lukaszkonieczny) [reply extensions/wikia/CreateNewWiki/crowdin-autocreate.conf:2]: Discussed offline ;)

**Review by idradm — APPROVED** [review]
